src/app/organizations/organization/organization.component.html

```diff
@@ -61,73 +67,88 @@ <h3>{{ org.displayName }}</h3>
-          *ngIf="org?.status === approved"
-          [organization]="org"
-          class="pull-right starring-button"
-          (change)="organizationStarGazersChange()"
-        ></app-organization-starring>
-      </mat-card-content>
       <mat-card-actions *ngIf="canEdit$ | async">
         <div fxFlex></div>
         <button mat-button (click)="editOrganization()" id="editOrgInfo" matTooltip="Edit the organization">
           <mat-icon>edit</mat-icon>
         </button>
       </mat-card-actions>
-    </mat-card>
+    </div>
 
     <!-- Tabs and description -->
-    <div fxLayout="row" fxLayout.lt-md="column" fxLayoutGap="15px" class="my-3" data-cy="organizationDetails">
+    <div fxLayout="row" fxLayout.lt-md="column" fxLayoutGap="2rem" class="my-3" data-cy="organizationDetails">
       <div fxFlex fxGrow="1" fxFlex.lt-md="100%">
         <mat-tab-group>
-          <mat-tab label="Collections">
+          <mat-tab>
+            <ng-template mat-tab-label>
+              <div fxLayoutGap="0.5rem">
+                <img src="../../../assets/svg/collections.svg" alt="collections icon" />
+                <span>Collections</span>
+                <span class="tab-chip">{{ getObjectKeys(collectionsQuery.collections$ | async).length }}</span>
+              </div>
+            </ng-template>
             <collections [organizationID]="org.id" [organizationName]="org.name"></collections>
           </mat-tab>
-          <mat-tab label="Members">
+          <mat-tab>
+            <ng-template mat-tab-label>
+              <div fxLayoutGap="0.5rem">
+                <mat-icon class="tab-icons">people</mat-icon>
+                <span>Members</span>
+                <span class="tab-chip">{{ (organizationMembersQuery.sortedOrganizationMembers$ | async).length }}</span>
+              </div>
+            </ng-template>
             <organization-members></organization-members>
           </mat-tab>
-          <mat-tab label="Events">
+          <mat-tab>
+            <ng-template mat-tab-label>
+              <div fxLayoutGap="0.5rem">
+                <mat-icon class="tab-icons">update</mat-icon>
+                <span>Updates</span>
+                <span class="tab-chip">{{ (eventsQuery.organizationEvents$ | async).length }}</span>
+              </div>
+            </ng-template>
             <events [organizationID]="org.id"></events>
           </mat-tab>
         </mat-tab-group>
       </div>
       <div fxFlex fxGrow="1" fxFlex.lt-md="100%">
-        <mat-card fxLayout="column" fxLayoutGap="15px" class="my-3">
+        <div fxLayout="column" fxLayoutGap="15px" class="my-3">
+          <h4>About the Organization</h4>
+          <div fxLayout="column" fxLayoutGap="10px" class="px-4 py-1">
+            <span fxLayout="row" fxLayoutGap="10px" *ngIf="org?.email">
+              <mat-icon class="side-icons">email</mat-icon>
+              <span
+                ><a [href]="'mailto:' + org.email" target="_top">{{ org.email }}</a></span
```

Comment:
Is the email required? I don't think it is. If it's not, we should handle more gracefully.

Reply:
@coverbeck I think this was there before, I will get rid of it?